src/sparseml/optim/manager.py

```diff
@@ -561,6 +529,44 @@ def qat_active(self, epoch: float) -> bool:
             else False
         )
 
+    def get_start_end_epochs(self) -> Dict[str, Tuple[float, float]]:
+        """
+        Return an OrderedDict mapping each stage to its min and max epoch. If not a
+        staged manager, map 'all' to the the min and max epochs
+        """
+        if isinstance(self.modifiers, List):
+            return OrderedDict({"all": (self.min_epochs, self.max_epochs)})
+        else:
+            stage_max_min = OrderedDict()
+            for stage, mod_list in self.modifiers.items():
+                epoch_min = _min_modifier_epoch(mod_list)
+                epoch_max = _max_modifier_epoch(mod_list)
+                stage_max_min[stage] = (epoch_min, epoch_max)
```

Comment:
This is absolutely not a blocker, especially cause this is a cherry-pick, but I feel `stage_min_max` is a better name for this variable

Reply:
That's a good point. Will be addressed on main, but will keep as is for release